src/main/java/org/session4/homework/Main.java

```diff
@@ -0,0 +1,23 @@
+package org.session4.homework;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class Main {
+    public static void main(String[] args) {
+
```

Comment:
Please don't leave an empty row right at the beginning of a method's body. We do that only for classes.